### DISCUSSION TITLE
GitHub's Public Key

### DISCUSSION THREAD

**Discussion by willbarkoff** [community/community#21586] [New to GitHub]
<p>Hi there!</p>
<p>This is my first time posting here, so please let me know if I’m doing something wrong. Thanks :slight_smile:.</p>
<p>I often use the <code>git</code> cli rather than the GitHub website to verify commits. Sometimes, I’ll either use the GitHub website to make a quick, one-line fix, or I’ll be working with someone who uses the website to write some code, and I’ll need to verify their commit. No problem, I think to myself, preparing to use <code>git verify-commit</code> as I always do.</p>
<pre><code class="lang-bash">$ git verify-commit &lt;hash&gt;
gpg: Signature made Wed May 19 20:23:48 2021 EDT
gpg:                using RSA key 4AEE18F83AFDEB23
gpg: Can't check signature: No public key
</code></pre>
<p>Uh oh! I can’t verify the commit because I don’t have GitHub’s public key. I was wondering if it is published anywhere, or if I can get it somewhere to do these verifications with the command line?</p>
<p>Thank you so much!</p>

**Comment by canuckjacq** [comment]
<p>:wave: Welcome!</p>
<p>I’ve seen this error before where the commit email address didn’t match the email address associated with the gpg key. Can you double check that?</p>

**Comment by graingert** [comment, marked as answer]
<p>the key is here <a href="https://github.com/web-flow.gpg" rel="noopener nofollow ugc">https://github.com/web-flow.gpg</a></p>
